test/verify/check-shell-host-switching

```diff
@@ -75,6 +76,9 @@ class HostSwitcherHelpers:
         if pixel_label:
             b.assert_pixels("#hosts_setup_server_dialog", pixel_label)
         b.click('#hosts_setup_server_dialog .pf-m-primary:contains("Add")')
+        if expect_warning:
+            b.wait_visible('#hosts_connect_server_dialog')
+            b.click('#hosts_connect_server_dialog button.pf-m-warning')
```

Comment:
Please wait for the dialog to go away, otherwise we risk mis-clicks with Firefox again.